src/main/webapp/WEB-INF/deployerConfigContext.xml

```diff
@@ -54,12 +51,106 @@
           c:authenticator-ref="authenticator">
         <property name="principalAttributeMap">
             <map>
-                <entry key="relayGuid" value="id" />
+            </map>
+        </property>
+    </bean>
+
+    <bean id="attributeRepository"
+          class="org.jasig.services.persondir.support.ldap.LdaptivePersonAttributeDao"
+          p:connectionFactory-ref="ldap.connection.factory.pooled"
+          p:baseDN="ou=sso,ou=account,dc=ccci,dc=org"
+          p:searchControls-ref="searchControls"
+          p:searchFilter="cn={0}">
+        <property name="resultAttributeMapping">
+            <map>
+                <!--
+                   | Key is LDAP attribute name, value is principal attribute name.
+                   -->
+                <entry key="relayGuid" value="ssoGuid" />
                 <entry key="cn" value="email" />
+                <entry key="givenName" value="first" />
+                <entry key="sn" value="last" />
```

Comment:
also, `firstName` and `lastName` I was thinking those were different than what you had, but wasn't completely sure until I just checked